appveyor.yml

```diff
@@ -1,9 +1,23 @@
-image: Visual Studio 2017
+image: 
+  - Visual Studio 2017
+  - Ubuntu
+  
+skip_tags: false
 
-before_build:
-  - nuget restore
+init:
+  - ps: >-
+      if ($env:APPVEYOR_REPO_TAG -eq "true")
+      {
+          Update-AppveyorBuild -Version ($env:APPVEYOR_REPO_TAG_NAME).TrimStart("v")
+      }
 
-build:
-  verbosity: minimal
-  project: NVelocity.sln
-  publish_nuget: false
+build_script:
+  - dotnet build ./NVelocity.sln -c Release 
```

Comment:
I read through the Core project and it seemed a little bit overkill (for now) to pull everything into a separate build file, and I was having trouble running it on Ubuntu. So I just put it in here for now.